addon/components/polaris-badge.hbs

```diff
@@ -1,6 +1,7 @@
+{{!-- TODO #polaris-v5-newDesignLanguage add WithinFilterContext utility and style --}}
```

Comment:
@tomnez I've merged #678 which adds support for this new design language option.
Let's update this and remove the TODO before considering this done